orcid-message-listener/src/main/resources/orcid-message-listener-web-context.xml

```diff
@@ -86,4 +80,40 @@
 		<property name="defaultDestination" ref="defaultJmsDestination" />
 	</bean>
 
+
+	<!-- Jersey client -->
```

Comment:
I was reviewing the servers and it looks we dont need this in any other env than DEV, but, it will not harm anyone if we leave it here